qcore/nhm.py

```diff
@@ -1,6 +1,34 @@
 from typing import List
+import datetime
 
 import numpy as np
+from scipy.stats import truncnorm
+
+from srf_generation.source_parameter_generation.uncertainties import mag_scaling
+
+
+NHM_HEADER = f"""FAULT SOURCES - New Zealand National Seismic Hazard Model 2010 (created {datetime.datetime.now().strftime("%d-%b-%Y")}) 
+Row 1: FaultName 
+Row 2: TectonicType , FaultType 
+Row 3: LengthMean , LengthSigma (km) 
+Row 4: DipMean , DipSigma (deg) 
+Row 5: DipDir 
+Row 6: Rake (deg) 
+Row 7: RupDepthMean , RupDepthSigma (km) 
+Row 8: RupTopMean, RupTopMin RupTopMax  (km) 
+Row 9: SlipRateMean , SlipRateSigma (mm/yr) 
+Row 10: CouplingCoeff , CouplingCoeffSigma (mm/yr) 
+Row 11: MwMedian , RecurIntMedian  (yr) 
+Row 12: Num Locations on Fault Surface 
+Row 13+: Location Coordinates (Long, Lat) 
+"""
+
+
+MU = 3.0 * 10.0 ** 10.0
```

Comment:
Maybe add a comment or rename variable so that its clear what MU this is? Based on usage looks like moment magnitude mu?